.setup/install_system.sh

```diff
@@ -558,14 +559,34 @@ if [[ ${VAGRANT} == 1 ]]; then
     # Disable OPCache for development purposes as we don't care about the efficiency as much
     echo "opcache.enable=0" >> /etc/php/7.0/fpm/conf.d/10-opcache.ini
 
+    #
+    # FIXME: commented out since symlink permissions aren't allowing
```

Comment:
@MasterOdin If you would like to fix the symlink permissions problem before merging, please help.  The logs are broken in master also.